folder_structure.md

```diff
@@ -254,6 +255,28 @@ The files are all produced by the [profiling-recipe](https://github.com/jump-cel
 
 - `2021_04_26_Batch1` is the batch and `BR00117035` is the plate
 
+### `embeddings` folder structure:
+
+The structure here is very similar to `analysis`
+
+```
+└── embeddings
```

Comment:
Probably the first question to ask: is this folder structure inherited from some other software?

Reply:
Also, do you know if Pycytominer follows similar folder structure? @shntnu

Reply:
It's inherited from our past selves :) I've adapted it from the way we do things in https://cytomining.github.io/profiling-handbook/